app/controlplane/pkg/authz/authz.go

```diff
@@ -153,6 +170,36 @@ var rolesMap = map[Role][]*Policy{
 		PolicyArtifactUpload,
 		// + all the policies from the viewer role inherited automatically
 	},
+	RoleOrgMember: {
+		// Allowed endpoints. RBAC will be applied where needed
+		PolicyWorkflowRead,
+		PolicyWorkflowContractRead,
+		PolicyWorkflowContractCreate,
+
+		PolicyWorkflowList,
+		PolicyWorkflowCreate,
+		PolicyWorkflowUpdate,
+		PolicyWorkflowDelete,
+
+		PolicyWorkflowRunList,
+	},
+	// RoleProjectAdmin: RBAC will be applied in all these
+	RoleProjectAdmin: {
+		// attestations
+
+		PolicyWorkflowRead,
```

Comment:
I do not think you need to duplicate the policies but instead make admin a superset of member and in here just add the things that it can do that others can't

Reply:
The one above is ORGMember, and those roles are just for the middleware to allow executing those endpoints